docs/README.md

```diff
@@ -460,6 +460,22 @@ Returns the diagnostic information associated with the most recently called func
 
 **Returns**: `Promise` when resolved returns `String` or the promise is rejected.
 
+## **Statement.enableNumericTypeConversion(flag)**
+
+Enables or disables automatic numeric conversion.
+
+**Parameters**:
+
+- **flag:** `boolean` to turn automatic data conversion mode on or off. Default value is `false`.
+
+    - `true`: The result data is returned as its original type. Due to the data type difference between SQL and Javascript, the data may be truncated.
```

Comment:
```suggestion
    - `true`: SQL numeric types (`INTEGER`, `DOUBLE`, `DECIMAL`, `DECFLOAT`, etc) are converted to Javascript `Number` objects instead of strings. Due to the data type limitations of Javascript `Number`s, precision may be lost. If the SQL value is outside the bounds of what a `Number` can represent, it will be returned as a string instead.
```

I'm not sure if that's how you're supposed to refer to Javascript Numbers in JSDoc.